subprojects/core/src/main/java/org/gradle/api/internal/tasks/TaskStateInternal.java

```diff
@@ -122,4 +123,12 @@ public boolean getNoSource() {
     public boolean isFromCache() {
         return outcome == TaskExecutionOutcome.FROM_CACHE;
     }
+
+    public boolean isActionsWereExecuted() {
```

Comment:
The `is` prefix is intended to indicate the boolean return type, but I don't feel strongly about this and can change it if you do.